Libraries/CMSIS/Device/Maxim/MAX32572/Source/GCC/max32572.ld

```diff
@@ -73,77 +57,40 @@ SECTIONS {
         KEEP(*(.init))
         KEEP(*(.fini))
 
-        /* .ctors */
-        *crtbegin.o(.ctors)
-        *crtbegin?.o(.ctors)
-        *(EXCLUDE_FILE(*crtend?.o *crtend.o) .ctors)
-        *(SORT(.ctors.*))
-        *(.ctors)
-
-        /* .dtors */
-        *crtbegin.o(.dtors)
-        *crtbegin?.o(.dtors)
-        *(EXCLUDE_FILE(*crtend?.o *crtend.o) .dtors)
-        *(SORT(.dtors.*))
-        *(.dtors)
```

Comment:
ctors and dtors are needed for C++ support, any idea why they were removed?

Reply:
This was created roughly 1-2 years ago, and has been kept separate/out of sync with the main as development was on and off.